samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -241,5 +451,13 @@ def _load_saved_pipeline_user_arn() -> Optional[str]:
     return config.get("pipeline_user")
 
 
+def _load_config_values() -> Dict[str, str]:
+    samconfig: SamConfig = SamConfig(config_dir=PIPELINE_CONFIG_DIR, filename=PIPELINE_CONFIG_FILENAME)
+    if not samconfig.exists():
+        return {}
+    config: Dict[str, str] = samconfig.get_all(cmd_names=_get_bootstrap_command_names(), section="parameters")
```

Comment:
Why we are loading parameters like this [instead of this](https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/sync/command.py#L97)?

Reply:
We do have this at line 59. This loading parameters part was used for some conditional OIDC config update. I tried using `@configuration_option` only but it doesn't work as we expect.

Reply:
we are duplicating this? I'm still not sure why its stated as not working.